0.download_data/scripts/download_plates.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Download NF1 datasets from Figshare
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+from download_figshare import download_figshare
+
+
+# ## Set constant paths/variables
+
+# In[2]:
+
+
+# figshare url for both plates
+figshare_url = "https://figshare.com/ndownloader/articles/"
+
+# metadata folder for metadata files from both plates to be moved into
+metadata_dir = pathlib.Path("metadata")
+
+# Since the Figshare download is a zip file for the NF1 data, we have to have the unzip_file parameter turned on
+unzip_files = True
+
+
+# ## Set dictionary with specific path/variables for each plate
+
+# In[3]:
+
+
+download_plates_info_dictionary = {
+    "Plate_1": {
+        "figshare_id": "22233292",
+        "version_number": "1",
+        "output_folder": "Plate_1_zip",
```

Comment:
is this a temp output folder? It's deleted?

Reply:
In our case, it is a temporary "folder" since Figshare downloads a zip file instead of just downloading all of the files into the `output_folder`.

I have changed the parameters to make `output_dir` optional and only used when dealing with zip files because normally the files should be automatically downloaded in the folder name specified in `output_folder`.